packages/codefixes/src/7006/index.ts

```diff
@@ -1,91 +0,0 @@
-import { findNodeAtPosition, insertIntoText } from '@rehearsal/utils';
-import { getDefaultFormatCodeSettings } from 'typescript';
-
-import { type FixedFile, FixTransform } from '../fix-transform';
-import type { DiagnosticWithLocation, TextChange } from 'typescript';
-import type { RehearsalService } from '@rehearsal/service';
-
-export class FixTransform7006 extends FixTransform {
-  fix = (diagnostic: DiagnosticWithLocation, service: RehearsalService): FixedFile[] => {
-    const node = findNodeAtPosition(diagnostic.file, diagnostic.start, diagnostic.length);
-
-    if (!node) {
-      return [];
-    }
-
-    const languageService = service.getLanguageService();
-    const formatOptions = getDefaultFormatCodeSettings();
-    const sourceFile = diagnostic.file;
-
-    const fileName = sourceFile.fileName;
-
-    const fixedFiles: FixedFile[] = [];
-
-    const fixes = languageService.getCodeFixesAtPosition(
-      sourceFile.fileName,
-      diagnostic.start,
-      diagnostic.start + diagnostic.length,
-      [7006],
-      formatOptions,
-      {}
-    );
-
-    const stubLocation = { startLine: 1, startColumn: 1, endLine: 1, endColumn: 1 }; // TODO: actually get the correct coordinates for this.
-
-    fixes.forEach((fix) => {
-      if (fix.fixId) {
-        // Why do we need a combined change?
-        const combined = languageService.getCombinedCodeFix(
```

Comment:
Tripple-checked :)
We don't need to use getCombinedCodeFix in our implementation of our plugin where we do fix error by error.
 
 getCombinedCodeFix returns fixes of the same kind per file... in IDE it is called "Fix all ..."